ui-design-lib/src/main/java/co/electriccoin/zcash/ui/design/theme/ZcashTheme.kt

```diff
@@ -27,22 +27,16 @@ fun ZcashTheme(
     content: @Composable () -> Unit
 ) {
     val useDarkMode = forceDarkMode || isSystemInDarkTheme()
+    val baseColors = if (useDarkMode) DarkColorPalette else LightColorPalette
+    val extendedColors = if (useDarkMode) DarkExtendedColorPalette else LightExtendedColorPalette
+    val zashiColors = if (useDarkMode) DarkZashiColorPalette else LightZashiColorPalette
+    val exchangeRateColors = if (useDarkMode) DarkExchangeRateColorPalette else LightExchangeRateColorPalette
```

Comment:
If we follow the comments above, we can then remove `exchangeRateColors` here.

Reply:
done